src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePropertiesSplittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const stylePropertiesSplittedByDoubleDot = stylePropertiesSplittedByDotComa
+    .map((item) => item.split(':'))
+    .map((SplittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      SplittedCssProperty.map((propertyPart) => propertyPart.trim()));
+
+  const cssPropertiesObject = [...stylePropertiesSplittedByDoubleDot].reduce(
```

Comment:
You don't need to create a copy as reduce doesn't mutate an array

Reply:
okaaay, Buddy was wrong.
fixed